src/System.Diagnostics.TraceSource/src/System/Diagnostics/DefaultTraceListener.cs

```diff
@@ -41,6 +41,7 @@ public override void Write(string message)
                     base.Write(message.Substring(offset));
```

Comment:
> really huge messages mess up both VS and dbmon, so we chop it up int

This logic is in DebugProvider baseclass already. We should not need to duplicate it here.

Reply:
I agree. I noticed this was already in DebugProvider when doing the refactoring and it had been duplicated here previously.

I was debating on whether or not to remove this.

Will remove ASAP.